pkgs/development/python-modules/plotly/default.nix

```diff
@@ -66,7 +65,7 @@ buildPythonPackage rec {
     ipython
     ipywidgets
     which
-    orca
+    # orca # unable to locate binary, is this the right package?
```

Comment:
Place `orca` and `psutil` in `optional-dependencies` per https://plotly.com/python/getting-started/#orca

Then take them out of the check inputs

Reply:
Also add `kaleido` to the check inputs and disable kaleido tests on Darwin (as they try to launch it as a subprocess). See my recommended changes on the previous PR.

Reply:
FYI The reason orca isn't working as a check input here is one of its dependencies isn't building on Darwin (libaio, which is Linux-specific.)

Reply:
nice finding that orca is legacy! 

`optional-dependencies` is supposed to mirror the optional deps [defined in the package manifest](https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-and-requirements), which I can't see plotly doing that neither in the code nor in the produced METADATA file.

kaleido is already in `dependencies` and thus need not also be added to check inputs

Reply:
They list the optional dependencies in https://github.com/plotly/plotly.py/blob/master/packages/python/plotly/optional-requirements.txt

You have more history with this, so I'll trust your choice.

Reply:
seem to have glossed over that one. while not a proper `[extras]` group, it is nice to document